src/commands/keyset.rs

```diff
@@ -1323,6 +1474,144 @@ struct KeySetConfig {
 
     /// Automatically remove keys that are no long in use.
     autoremove: bool,
+
+    /// KMIP servers to use, keyed by user chosen HSM id.
+    kmip_servers: HashMap<String, KmipServerConnectionSettings>,
```

Comment:
I wonder if this should move to KeySetState. Signers read KeySetState to figure out which keys to sign with. I think it would be nicer if all information is there. Otherwise we have to have a standard file format to KeySetConfig as well.

Reply:
@Philip-NLnetLabs and I discussed this and agreed to move the KMIP config into the `KeyStateState` because that is all that "downstreams" such as `dnst signer` have available to them.